compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMCPUTarget.cpp

```diff
@@ -843,21 +843,9 @@ void registerLLVMCPUTargetBackends(
     return std::make_shared<LLVMCPUTargetBackend>(queryOptions());
   };
 
-  // Preferred name.
+  // #hal.device.target<"llvm-cpu", ...
   // #hal.executable.target<"llvm-cpu", ...
-  static TargetBackendRegistration registration0("llvm-cpu", backendFactory);
-
-  // Abbreviated names.
-  // #hal.device.target<"cpu", ...
-  static TargetBackendRegistration registration1("cpu", backendFactory);
```

Comment:
@powderluv I know you expressed a desire to keep the `cpu` abbreviation, but the name is ambiguous (vmvx is also cpu)
https://github.com/iree-org/iree/pull/9854#discussion_r926098559

Reply:
yeah, we need to drop it - wasm is also cpu, and the new inline execution modes are also cpu